packages/ibm-products/src/components/Card/CardHeader.tsx

```diff
@@ -118,7 +132,7 @@ export const CardHeader = ({
       <div
         className={cx([
           `${headerClass}-container`,
-          { [`${headerClass}-container--has-slug`]: !!slug },
+          { [`${headerClass}-container--has-slug`]: !!aiLabel || !!slug },
```

Comment:
```suggestion
          { [`${headerClass}-container--has-aiLabel`]: !!aiLabel || !!slug },
```

Reply:
yes you're absolutely correct. good call